Source/Scene/BillboardRenderTechnique.js

```diff
@@ -0,0 +1,37 @@
+/*global define*/
+define([
+    '../Core/freezeObject'
+], function(
+    freezeObject) {
+    'use strict';
+
+    /**
+     * Determines how billboards and points are rendered.
+     *
+     * @exports BillboardRenderTechnique
+     */
+    var BillboardRenderTechnique = {
```

Comment:
Given that this applies to points and labels as well, I don't think it should have `Billboard` in the name.  I am also hesitant about having `technique` in the name since in glTF and computer graphics, this isn't really a technique in the material sense.

Perhaps throughout, this is better named `BlendOption` (too confusing with blend mode? probably not at this level of the API), `ImageBlendOption`, `RenderPasses` (again, too similar to the renderer?), or `CompositeMode`?  Ignore the implementation and think about what would sound reasonable to the end user.